CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ x.x.x
 
 **BUG FIXES**
 - Fix default for disable validate and test components when building custom AMI. The default was to disable those components, but it wasn't effective.
+- Fix `export-cluster-logs` command filter option.
```

Comment:
Can we provide more context here to what we're fixing?

Reply:
Yes, changed to: Fix unable to export cluster's logs error when using `export-cluster-logs` command with `--filters` option.

Before the fix, the error message is:
pcluster export-cluster-logs -n slurm-memory --bucket test-bucket --filters "Name=node-type,Values=HeadNode"

ERROR: Unable to export cluster's logs.
Unexpected error when exporting cluster's logs: Invalid filters Name=node-type,Values=HeadNode. They must be in the form Name=...,Values=...